server/src/main/java/com/objectcomputing/checkins/services/memberprofile/MemberProfile.java

```diff
@@ -57,20 +57,21 @@
 
     @Column(name="startDate")
     @Schema(description = "employee's date of hire", required = true)
+    @Nullable
```

Comment:
Schema says its required, but you have it as nullable. Which one is it?

Reply:
This is something that could need to be entered still by the time a person jumps into the app. We should allow this to be nullable.

Reply:
@chrismckiernan good catch. I've updated this in the latest commit.